### PR TITLE
Adding admin approval for certificate changes. (CP: #8859)

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Cherry-pick of 2.0's #8859.

Adding admins as required approvers of modifications to the set of trusted CAs.

###### Does this affect the toolchain?  

No.

###### Test Methodology

- Local tests with `.gitignore`.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -87,6 +87,9 @@
 /toolkit/scripts/toolchain/cgmanifest.json @microsoft/cbl-mariner-admins
 /toolkit/scripts/toolchain/create_toolchain_in_container.sh @microsoft/cbl-mariner-admins
 
+# Modifications to the trusted CA certificates require admin approval.
+/SPECS/*ca-certificates*/*
```
[the remark this answers is not in the document]
Comment (PawelWMS) [reply .github/CODEOWNERS]: Correct, I must have accidentally removed the group. Thank you, @mfrw. And it looks like I'll need to fix `main` as well.

[the remark this answers is not in the document]
Comment (PawelWMS) [reply .github/CODEOWNERS]: There we go: #8895.

**Review by mfrw — APPROVED** [review]
LGTM :)

File: .github/CODEOWNERS (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfrw) [reply .github/CODEOWNERS]: > There we go: #8895.

Approved 🥇

**Review by anphel31 — APPROVED** [review]

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**